js/src/forum/components/PostStream.js

```diff
@@ -73,7 +75,7 @@ export default class PostStream extends Component {
       }
 
       return (
-        <div className="PostStream-item" {...attrs}>
+        <div className="PostStream-item fade-in" {...attrs}>
```

Comment:
This seems to be a left-over, right? We extracted the CSS changes to master in #2262, but even that doesn't even have this CSS class... :shrug: